native/cocos/editor-support/spine/CurveTimeline.h

```diff
@@ -45,8 +45,6 @@ class SP_API CurveTimeline : public Timeline {
 
     virtual void apply(Skeleton& skeleton, float lastTime, float time, Vector<Event*>* pEvents, float alpha, MixBlend blend, MixDirection direction) = 0;
 
-    virtual int getPropertyId() = 0;
```

Comment:
getPropertyId is defined in Timeline, doesn't need to define the pure virtual function here again.